CHANGELOG.rst

```diff
@@ -0,0 +1,14 @@
+=====================================
+Eclipse_Bluechi.Bluechi Release Notes
```

Comment:
I removed this file as it will be automatically generated during the release (see the release.yml workflow). I have updated `config.yaml` file so that it will generate the title according to your comment